Casks/cinny.rb

```diff
@@ -0,0 +1,26 @@
+cask "cinny" do
+  version "4.0.3"
+  sha256 "1aff22fa0b20b82daf5ab578ed4ed658311226bc5417081a0f4af5d32a84c446"
```

Comment:
Will the cask be automatically updated or we will have to do it manually? Also will it use the built-in auto update or something else?
If not can we setup something to update the cask file automatically?

Reply:
Can we please setup auto-update along with this addition as we want to automate as much release workflow we can.

Reply:
Sure! Added the GitHub Action, let me know if anything else is needed.